recipes/libmetatensor-torch/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.5.0" %}
+{% set url_base = "https://github.com/lab-cosmo/metatensor/releases/download" %}
+
+package:
+  name: libmetatensor-torch
+  version: {{ version }}
+
+source:
+  url: {{ url_base }}/metatensor-torch-v{{ version }}/metatensor-torch-cxx-{{ version }}.tar.gz
+  sha256: 904cf858d8f98b67b948e8a453d8a6da56111e022050d6c8c3d32a9a2cc83464
+
+build:
```

Comment:
Shared libraries also need a run_exports.

Reply:
It's not very clear what run_exports does. Does it only add a `run` dependency to all packages that built against the library and had it as a `host` dependency? Or is there something else I need to be aware of here?

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements

run_exports export runtime requirements to downstream packages which dynamically link to your package. It's a statement which says "hey, if you link to my package, then this is the version that you will need at runtime."

Reply:
Thanks! I've added `run_exports` to [`libmetatensor`](https://github.com/conda-forge/libmetatensor-feedstock) and removed it from the `run` requirements here. 

I'd guess `libtorch` also has a run_exports, but is there a way to express "build against the cpu variant but run with any variant, cpu or cuda"? For now I do two separate host & run requirements.

Reply:
> I'd guess libtorch also has a run_exports

[Yes.
](https://github.com/conda-forge/pytorch-cpu-feedstock/blob/2953d76cb7cfa5fd5b8049c1df9baf0758bab4af/recipe/meta.yaml#L43)

Looks like the run_exports for `libtorch` are variant agnostic, so if you build against any variant of `libtorch` you can get any variant at runtime. It looks like this package doesn't link against `libtorch_cuda`, so you don't need to worry about contraining the `libtorch` build variant.